client/pages/LoginPage.jsx

```diff
@@ -26,9 +26,10 @@ function LoginPage ({user, setUser}) {
             // Eventually add this functionality to route to home page displaying all boards
             //navigate('/boards')   // user.board_ids   // getBoards from backend - boardNames and Id
             // for now we just directly go to the first board
-            navigate(`/boards/${userData.board_ids[0]}`);
+            // navigate(`/boards/${userData.board_ids[0]}`);
+            navigate(`/boards`);
```

Comment:
If we make the decision to navigate to `/boards` instead, let's remove the commented out code in L29

Reply:
Agreed. I think once I fully implement the boards display page and routing, then I will remove all of those relevant comments